dev/make-distribution.sh

```diff
@@ -201,6 +201,12 @@ fi
 # Copy data files
 cp -r "$SPARK_HOME/data" "$DISTDIR"
 
+# Make pip package
+echo "Building python distribution package"
+cd $SPARK_HOME/python
+python setup.py sdist
```

Comment:
Add a parameter so we only build this if the parameter is given? I think we may not need to build this by default.